sql/migrations/20241209235857_v52.7.sql

```diff
@@ -0,0 +1,2 @@
+-- Modify "WorkerAssignEvent" table
+ALTER TABLE "WorkerAssignEvent" DROP CONSTRAINT "WorkerAssignEvent_workerId_fkey", ALTER COLUMN "workerId" DROP NOT NULL, ADD CONSTRAINT "WorkerAssignEvent_workerId_fkey" FOREIGN KEY ("workerId") REFERENCES "Worker" ("id") ON UPDATE CASCADE ON DELETE SET NULL NOT VALID;
```

Comment:
Just to confirm, have we tested this migration against a database which has existing Worker and WorkerAssignEvents in the table?

Reply:
yes seems fine - tested on cloud copy with 
cloud=> select count(*) from "WorkerAssignEvent";
  count
---------
 5307060
(1 row)

select count(*) from "Worker";
 count
-------
 92610
(1 row)